utils/test/svg-mock.js

```diff
@@ -5,8 +5,10 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import React from 'react';
+import styled from 'styled-components';
 
-const MockSVG = props => <span {...props}>SVG</span>;
+const MockSVG = styled.svg.attrs(() => ({
```

Comment:
Why is `styled-components` necessary for this? Could we just use a standard component. Like:

```jsx
const MockSVG = props => <svg xmlns="http://www.w3.org/2000/svg" {...props} />;
```

Reply:
Yeah, had it that way originally and then decided to be fancy. Reverting.